static/src/javascripts/projects/common/modules/commercial/creatives/fluid250.js

```diff
@@ -4,8 +4,7 @@ define([
     'common/utils/$',
     'common/utils/mediator',
     'common/utils/storage',
-    'common/utils/template',
```

Comment:
Ahh, I see. If that's the only usage could we remove `common/utils/template` and just require `lodash/utilities/template` directly?